AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -70,6 +70,7 @@ biershop-bayern.de,nkd.com,4f.com.pl,alcatelmobile.com,altex.ro,berliner-zeitung
 !
 1und1.de###__tealiumGDPRcpPrefs
 dak.de##.notification[data-handler-notification^='{"case": "cookie"']
+essen-und-trinken.de##div[class^="sp_message_container"]
```

Comment:
breaks scrolling

Reply:
> breaks scrolling

`essen-und-trinken.de#@#.googleAd`
Adblock detection don't work because of this rule => scrolling doesn't break